<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>810</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1300</width>
    <height>810</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>3d Scanner - V0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_MainLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="txtProgramLog">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ClearLog">
       <property name="text">
        <string>Clear Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tab_Menu">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Camera</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>161</width>
           <height>128</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="lblTabCameraControls">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="statusTip">
             <string>Control the main camera (left)</string>
            </property>
            <property name="text">
             <string>Controls</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_PauseOrResume">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_PrintImgData">
            <property name="text">
             <string>Print Img Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_SaveImage">
            <property name="text">
             <string>Take Screenshot</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Scanner</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>168</width>
           <height>181</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lblTabScannerControls">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="statusTip">
             <string>Control the main camera (left)</string>
            </property>
            <property name="text">
             <string>Controls</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ChangeContrast">
            <property name="text">
             <string>Change Contrast</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_GrayScale">
            <property name="text">
             <string>Gray Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_InvertBits">
            <property name="text">
             <string>Invert Bits</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_CannyEdgeDetection">
            <property name="text">
             <string>Canny Edge Detection</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>0</y>
           <width>160</width>
           <height>128</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lbl_Alpha">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_ContrastSliderAlpha">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_Beta">
            <property name="text">
             <string>Beta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_ContrastSliderBeta">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_Treshold">
            <property name="text">
             <string>Treshold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_TresholdSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>0</y>
           <width>161</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="lblTabScannerControls_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="statusTip">
             <string>Control the main camera (left)</string>
            </property>
            <property name="text">
             <string>Scan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ScanImg">
            <property name="text">
             <string>Scan Colors</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
